qutebrowser/browser/downloads.py

```diff
@@ -131,7 +131,11 @@ def create_full_filename(basename, filename):
     encoding = sys.getfilesystemencoding()
     filename = utils.force_encoding(filename, encoding)
     basename = utils.force_encoding(basename, encoding)
-    if os.path.isabs(filename) and os.path.isdir(filename):
+
+    if filename.endswith('/'):
```

Comment:
Hmm, right - I'm not actually sure if all places handle this correctly, but Windows' APIs are usually fine with getting `/` instead of `\` (it's mostly only `cmd.exe` having trouble with `/`).

What about `if filename.endswith('/') or filename.endswith(os.sep):`?